tests/source/cargo-toml/basic/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+
+
+  # comment before [[bin]], not necessarily at the beginning
+
+
+    [[bin]]    # comment  after [[bin]]
+        "aa" = 1     # comment for "aa"    
```

Comment:
Would it be possible to move these tests to `tests/source/configs/format_cargo_toml/true` and adding the `#rustfmt-format_cargo_toml: true` comment I mentioned above.

To test `format_cargo_toml=false` we only need to add tests to `tests/target/config/format_cargo_toml/false` to check that we don't format anything when the config is false.